### ISSUE TITLE
Font configuration

### ISSUE THREAD

**Issue by cleverdevil** [tbodt/ish#386]
It would be nice to be able to configure the font that is used by iSH as a user preference.

**Comment by Ecleptic** [comment]
That would definitely be fantastic.

**Comment by tbodt** [comment]
I'm planning on waiting for the improved font management that's supposed to be coming in iOS 13.

**Comment by Rexogamer** [comment]
bump

**Comment by kkk669** [comment]
@tbodt You've specified `UIFontDescriptorTraitMonoSpace` in https://github.com/tbodt/ish/commit/4f705ce2de0999419a6a5d3910ba3b3e91a92c6b#diff-44fefd97330f859e76143605b4824893R139 and I believe you are right. However, three monospaced fonts I installed on my device are not shown in that case. If I tried to delete the line and build it, all my fonts were shown. Any workaround?

![7C207086-C8F3-449A-ABAB-6916A2CD98D8](https://user-images.githubusercontent.com/601636/68108864-659de000-ff2c-11e9-89b9-aff476559ff5.jpeg)
